package exceptiontopic;

import java.util.Scanner;

public class TestApp1 
{
	public static void main(String[] args) 
	{
/*		int no1,no2,ans;
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter No1 : ");
		no1=  sc.nextInt();
		System.out.println("Enter No2 : ");
		no2=  sc.nextInt();
		ans = no1 / no2;// code crashed
		System.out.println("Division : " + ans);
		-------------------------------------------------------
		String name = null;
		System.out.println("name.length() : " + name.length());
		-------------------------------------------------------
		//             01234
		String name = "royal";
		System.out.println(name.charAt(5));// r
		-------------------------------------------------------
		int a[] = new int[5];//a[0],a[1],a[2],a[3],a[4]
		a[5] = 100;
		System.out.println("a[5] : " + a[5]);
		-------------------------------------------------------
		String value = "1a2b3";
		int no = Integer.parseInt(value);
		System.out.println("No : " + no);
		-------------------------------------------------------
		A objA = new B();
		C objC = (C)objA;
		-------------------------------------------------------
 */		
		
		
		
		
	}
}
